.github/workflows/ghcr.yml

```diff
@@ -0,0 +1,37 @@
+name: publish to ghcr
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+jobs:
+  push:
+    name: "g5v:latest"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - id: string
+        uses: ASzc/change-string-case-action@v2
+        with:
+          string: ${{ github.repository_owner }}
+      - uses: actions/checkout@v2
+      - uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: build
+      - uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: DockerfileLight
+          platforms: linux/amd64
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository_owner }}/g5v:latest
```

Comment:
Variable should be `${{ steps.string.outputs.lowercase }}`, or else that will fail :)

Reply:
Done